gallery/src/components/NxCollapsibleRadioSelect/NxCollapsibleRadioSelectPage.tsx

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxP, NxCode } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxCollapsibleRadioSelectExample from './NxCollapsibleRadioSelectExample';
+import NxCollapsibleRadioSelectCustomTooltipExample from './NxCollapsibleRadioSelectCustomTooltipExample';
+import NxCollapsibleRadioSelectDisabledExample from './NxCollapsibleRadioSelectDisabledExample';
+
+const NxCollapsibleRadioSelectExampleCode = require('./NxCollapsibleRadioSelectExample?raw'),
+    NxCollapsibleRadioSelectDisabledExampleCode = require('./NxCollapsibleRadioSelectDisabledExample?raw'),
+    NxCollapsibleRadioSelectCustomTooltipExampleCode =
+        require('./NxCollapsibleRadioSelectCustomTooltipExample?raw');
+
+const NxCollapsibleRadioSelectPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A collapsible items view radio group component.
```

Comment:
Could be explained better

Reply:
> Could be explained better

Updated: 24bcaf06aad89a58fa47b2a2b0e474070aad4bd0